test/functional/Java11andUp/playlist.xml

```diff
@@ -47,7 +47,7 @@
 	</test>
 
 	<test>
-		<testCaseName>StringvalueOfCodePoint_JDK11_CompactStrings_Disabled</testCaseName>
+		<testCaseName>String_JDK11_CompactStrings_Disabled</testCaseName>
```

Comment:
There is no need to separate the test into two `<test>` if the only difference is the variation. We can use one `<test>` and two `<variation>`.

```
<test>
   <testCaseName>String_CompactStrings</testCaseName>
   <variations>
      <variation>-XX:+CompactStrings</variation>
      <variation>-XX:-CompactStrings</variation>
   </variations>
   ...
```
Also, please do not have JDK number in the test name, in case we need to run this test in JDK11+ in the future.